### PR TITLE
Minor and random improvements

### PR DESCRIPTION
**Pull request by lfrancke**
- Update product list on the Homepage & Wording
- Update Playbooks
- Add a simple Contributors Guide
- Remove link to the Internal documentation

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,17 +1,14 @@
 = Stackable Documentation
 
 Welcome to Stackable!
-This documentation gives you an overview of our platform, how to install and manage it as well as some tutorials.
+This documentation gives you an overview of our Stackable Data Platform, how to install and manage it as well as some tutorials.
```
Comment (teozkr) [inline modules/ROOT/pages/index.adoc:4]:
```suggestion
This documentation gives you an overview of the Stackable Data Platform, how to install and manage it as well as some tutorials.
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -22,17 +19,19 @@ While the platform got started in the _Big Data_ ecosystem we are in no way limi
 
 You can declaratively build these environments, and we don't stop at the tool level as we also provide ways for the users to interact with the platform in the "as Code"-approach.
 
-We are leveraging the https://www.openpolicyagent.org/[OpenPolicyAgent] to provide Authorization-as-Code.
+We are leveraging the https://www.openpolicyagent.org/[OpenPolicyAgent] to provide Security-as-Code.
```
Comment (teozkr) [inline modules/ROOT/pages/index.adoc:22]:
```suggestion
We are leveraging the https://www.openpolicyagent.org/[Open Policy Agent] to provide Security-as-Code.
```

Security feels like a much wider concept than what OPA provides or interacts with.

File: modules/contributor/nav.adoc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+* Contributors Guide
```
Comment (teozkr) [inline modules/contributor/nav.adoc:1]:
```suggestion
* Contributor's Guide
```

File: modules/contributor/pages/development_dashboard.adoc [hunk 4]
```diff
@@ -0,0 +1,116 @@
+= Development Dashboard
+
+This page contains information about our products and relevant links you might need during development.
```
[the remark this answers is not in the document]
Comment (lfrancke) [reply modules/contributor/pages/development_dashboard.adoc:3]: Yes, this was me being lazy. This was mostly a copy&paste operation so far :)

I'll add those two

File: modules/contributor/nav.adoc [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+** xref:index.adoc[Welcome]
+** xref:development_dashboard.adoc[]
+** xref:new_version.adoc[]
```
[the remark this answers is not in the document]
Comment (lfrancke) [reply modules/contributor/nav.adoc]: Yes, I didn't find a way to do that though.
I'd like to include the adr/nav.adoc but that didn't work.
I'll leave it for now.
We might have to move the adr directory into the contributor one

[the remark this answers is not in the document]
Comment (teozkr) [reply modules/contributor/nav.adoc]: I think moving the ADRs into the `contributor` module makes sense.

[the remark this answers is not in the document]
Comment (lfrancke) [reply modules/contributor/nav.adoc]: Done, should be ready for review now

[the remark this answers is not in the document]
Comment (teozkr) [reply modules/contributor/nav.adoc]: I'm not seeing the ADRs in the PR, but it should probably be fine to punt on that for a later PR.

**Review by teozkr — DISMISSED** [review]

File: modules/contributor/nav.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (lfrancke) [reply modules/contributor/nav.adoc]: Hmmm I must have forgotten to push

**Review by teozkr — APPROVED** [review]

**Pull request merged**